test/output_test.dart

```diff
@@ -59,4 +59,34 @@ void main() {
       });
     });
   });
+
+  // Regression test for sass/dart-sass#417.
```

Comment:
I wouldn't call this a regression test, since it's really more of a feature.

Reply:
yep, copy-paste from above, removed "regression"